license-scan/Cargo.toml

```
@@ -1,10 +1,13 @@
 [package]
 name = "bottlerocket-license-scan"
 version = "0.1.0"
+description = "Collects and identifies licenses of Rust and Golang dependencies"
 license = "Apache-2.0 OR MIT"
 authors = ["iliana destroyer of worlds <iweller@amazon.com>"]
+repository = "https://github.com/bottlerocket-os/bottlerocket-sdk"
 edition = "2018"
```

```suggestion
edition = "2021"
```
Should we bump our edition and rust dependencies before we publish?

---

Yes, a dependency bump is definitely in order.

We don't need to increment the edition as a blocker here though, Rust allows editions to peacefully coexist.